packages/react-mutation-mapper/src/util/OncoKbUtils.ts

```diff
@@ -261,6 +261,19 @@ export function mergeAlterations(alterations: string | string[]) {
     return regular.join(', ');
 }
 
+export function getPositionalAlteration(alteration: string) {
+    const regExp = new RegExp('^(([A-Z]+)([0-9]+))([A-Z]*)$');
+    const result = regExp.exec(alteration);
+    if (
```

Comment:
maybe add a comment here what `result[2]`, `result[3]`, `result[4]` etc. correspond to? Maybe also add a brief comment on top of the function to summarize what we are doing here.

Reply:
@onursumer updated

Reply:
not necessary but check out Array destructuring: https://medium.com/the-andela-way/this-trick-will-help-you-de-structure-javascript-arrays-in-a-better-cleaner-way-a3bbc39ce226

Reply:
@alisman this is really cool! I will bare this in mind and use next time.